### PR TITLE
[ID] Updating learning environment minikube

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR is addressing issue in #26955 based on this PR #24533

/cc @kubernetes/sig-docs-id-reviews

### CODE REVIEW

File: content/id/docs/setup/learning-environment/_index.md [hunk 1]
```diff
@@ -2,3 +2,34 @@
...
+
+## kind
+
+[`kind`](https://kind.sigs.k8s.io/docs/) memberikan kamu kemampuan untuk
+menjalankan Kubernetes pada komputer lokal kamu. Perangkat ini membutuhkan
+[Docker](https://docs.docker.com/get-docker/) yang sudah diinstal dan
+terkonfigurasi.
+
+Halaman [Memulai Cepat](https://kind.sigs.k8s.io/docs/user/quick-start/) `kind`
+memperlihatkan kepada kamu tentang apa yang perlu kamu lakukan untuk `kind`
```
Comment (habibrosyad) [inline content/id/docs/setup/learning-environment/_index.md:14]:
```suggestion
memperlihatkan kepada kamu tentang apa yang perlu kamu lakukan agar `kind` dapat
```

File: content/id/docs/setup/learning-environment/_index.md [hunk 2]
```diff
@@ -2,3 +2,34 @@
...
+sehari-hari.
+
+Kamu bisa mengikuti petunjuk resmi
+[Memulai!](https://minikube.sigs.k8s.io/docs/start/) 
+`minikube` jika kamu ingin fokus agar perangkat ini terinstal.
+
+<a class="btn btn-primary" href="https://minikube.sigs.k8s.io/docs/start/" role="button" aria-label="Lihat Panduan Memulai! Minikube">Lihat Panduan Memulai! Minikube</a>
+
+Setelah kamu memiliki `minikube` yang bekerja, kamu bisa menggunakannya
+untuk  [menjalankan aplikasi contoh](/id/docs/tutorials/hello-minikube/).
```
Comment (habibrosyad) [inline content/id/docs/setup/learning-environment/_index.md:35]:
```suggestion
untuk [menjalankan aplikasi contoh](/id/docs/tutorials/hello-minikube/).
```

File: content/id/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ Untuk info lebih lanjut tentang perintah `docker build`, baca [dokumentasi Docke
 
     {{< kat-button >}}
 
-    {{< note >}}Kalau kamu memilih instalasi Minikube secara lokal,  jalankan `minikube start`.{{< /note >}}
+    {{< note >}}Kalau kamu memilih instalasi minikube secara lokal,  jalankan `minikube start`.{{< /note >}}
```
Comment (habibrosyad) [inline content/id/docs/tutorials/hello-minikube.md:57]:
```suggestion
    {{< note >}}Kalau kamu memilih instalasi minikube secara lokal, jalankan `minikube start`.{{< /note >}}
```

File: content/id/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -163,7 +163,7 @@ Supaya Kontainer `hello-node` bisa diakses dari luar jaringan virtual Kubernetes
 
 ## Aktifkan addons
 
-Minikube punya beberapa <i>addons</i> yang bisa diaktifkan, dinon-aktifkan, maupun dibuka di dalam <i>environment</i> Kubernetes lokal.
+Perangkat minikube meliputi sekumpulan {{< glossary_tooltip text="addons" term_id="addons" >}} bawaan yang bisa diaktifkan, dinon-aktifkan, maupun dibuka di dalam <i>environment</i> Kubernetes lokal.
```
Comment (habibrosyad) [inline content/id/docs/tutorials/hello-minikube.md:166]:
```suggestion
Perangkat minikube meliputi sekumpulan {{< glossary_tooltip text="addons" term_id="addons" >}} bawaan yang bisa diaktifkan, dinonaktifkan, maupun dibuka di dalam <i>environment</i> Kubernetes lokal.
```
no need for `-` (https://kbbi.web.id/nonaktif)

File: content/id/docs/tutorials/hello-minikube.md [hunk 5]
```diff
@@ -249,13 +249,13 @@ kubectl delete service hello-node
 kubectl delete deployment hello-node
 ```
 
-Kamu juga boleh mematikan mesin virtual (VM) untuk Minikube:
+Kamu juga boleh mematikan mesin virtual (VM) untuk minikube:
```
Comment (habibrosyad) [inline content/id/docs/tutorials/hello-minikube.md:252]: I don't have a strong opinion on this, but VM is actually abbreviation of _virtual machine_. Thus, imo abbreviating _mesin virtual_ with VM is a bit weird.

**Review by habibrosyad — COMMENTED** [review]
Thanks @ariscahyadi! I have few suggestions and comment for this PR.

**Comment by irvifa** [comment]
Hi @ariscahyadi I think this PR is good to go after you address the comments

File: content/id/docs/tutorials/hello-minikube.md (hunk 4 above)
Comment (ariscahyadi) [reply content/id/docs/tutorials/hello-minikube.md:166]: thanks @habibrosyad for the tips.

File: content/id/docs/tutorials/hello-minikube.md (hunk 5 above)
Comment (ariscahyadi) [reply content/id/docs/tutorials/hello-minikube.md:252]: I put like this `mesin virtual atau virtual machine (VM)`

**Comment by ariscahyadi** [comment]
@habibrosyad and @irvifa all issues have been addressed.

/lgtm

**Comment by habibrosyad** [comment]
👍
/lgtm

**Review by irvifa — COMMENTED** [review]
/lgtm

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**